services/frontend-service/src/ui/components/ServiceLane/ServiceLane.scss

```diff
@@ -59,6 +59,7 @@ Copyright freiheit.com*/
     align-self: center;
```

Comment:
please rephrase the commit message, e.g.
`feat(ui): Make commit bubble clickable`